### ISSUE TITLE
ModuleNotFoundError: No module named 'hololink.native._native'

### ISSUE THREAD

**Issue by jaydcarlson** [nvidia-holoscan/holoscan-sensor-bridge#1]
After installing the 1.1.0-EA release via `pip install ./python`, I attempt to run any example, but get the error:
```
ModuleNotFoundError: No module named 'hololink.native._native'
```

Older releases work without issue

**Comment by p-ogrady** [comment]
Sorry you're having trouble.  The error you're seeing is because the C++ part of the framework hasn't been properly set up.  If you follow the instructions in the user guide for [building the container](https://docs.nvidia.com/holoscan/sensor-bridge/1.0.0/setup.html#build-the-demo-container), do you still see problems?

**Comment by jaydcarlson** [comment]
I'm trying to build it natively without using containers. On prior Holoscan Sensor Bridge releases, I've been able to `pip install ./python` and it would automatically run the CMake build for the native components. Is that no longer supported?

**Comment by p-ogrady** [comment]
Holoscan sensor bridge is tightly coupled with Holoscan SDK, which is supported via containers; so  (at this time) building outside that environment isn't supported.

**Comment by p-ogrady** [comment]
(Closing this item; please reach out if you have any other questions!)

**Comment by jaydcarlson** [comment]
Just as an epilogue, I discovered this was actually my bad: while debugging another build issue I had turned off `HOLOLINK_BUILD_PYTHON` in CMakeLists.txt and completely forgot to turn it back on. Doh! Running `pip install ./python` now correctly builds the CMake-based native code and links it together into the Python module and installs it to the system.

_(So while you guys might not support running this outside of a container, it definitely works, and given the annoyances of doing development inside containers, this is definitely preferred for the bulk of development by my team.)_
